instrumentation-api-incubator/src/main/jflex/SqlSanitizer.jflex

```diff
@@ -358,7 +441,40 @@ WHITESPACE          = [ \t\r\n]+
   "NEXT" {
           if (!insideComment && !extractionDone) {
               extractionDone = operation.handleNext();
+          }
+          appendCurrentFragment();
+          if (isOverLimit()) return YYEOF;
+      }
+  "TABLE"  {
+          if (!insideComment && !extractionDone) {
+              if (operation.waitingOperationTarget()) {
+                extractionDone = operation.handleOperationTarget("TABLE");
+              } else {
+                extractionDone = operation.handleIdentifier();
+              }
+          }
+          appendCurrentFragment();
+          if (isOverLimit()) return YYEOF;
+      }
+  "INDEX"  {
+          if (!insideComment && !extractionDone) {
+            if (operation.waitingOperationTarget()) {
+              extractionDone = operation.handleOperationTarget("INDEX");
+            } else {
+              extractionDone = operation.handleIdentifier();
             }
+          }
+          appendCurrentFragment();
+          if (isOverLimit()) return YYEOF;
+      }
+  "DATABASE"  {
```

Comment:
these aren't the only things that could be created or dropped, any reason you chose to handle only these 3?

Reply:
Added missing ones with the reference to [SQL CREATE](https://www.w3schools.com/sql/sql_ref_create.asp).